.gitignore

```diff
@@ -32,7 +32,11 @@ junit.xml
 .pabotsuitenames
 /tests/robot-tests/backup-data
 /tests/robot-tests/webdriver
-/tests/robot-tests/.env.*
+/tests/robot-tests/.env
```

Comment:
If we're just looking to preserve the `.env.example`, then the following might be better:
```
/tests/robot-tests/.env.*
!/tests/robot-tests/.env.example
```

This prevents accidentally checking in envs for pre-prod and test for example.

Reply:
Oh that's pretty cool. Didn't know you could do that.